README.md

```diff
@@ -18,25 +20,30 @@ Contents:
 
 <!-- TOC depthFrom:2 depthTo:6 withLinks:1 updateOnSave:1 orderedList:0 -->
 
-- [Docker, Singularity and Dockstore](#docker-singularity-and-dockstore)
-- [Dependencies/Install](#dependenciesinstall)
-- [Creating a release](#creating-a-release)
-	- [Preparation](#preparation)
-	- [Cutting the release](#cutting-the-release)
-- [LICENCE](#licence)
+- [cgpPindel](#cgppindel)
+  - [Docker, Singularity and Dockstore](#docker-singularity-and-dockstore)
+  - [Dependencies/Install](#dependenciesinstall)
+  - [Creating a release](#creating-a-release)
+    - [Preparation](#preparation)
+    - [Release process](#release-process)
+      - [Code changes](#code-changes)
+      - [Docker image](#docker-image)
+      - [Cutting the release](#cutting-the-release)
+  - [LICENCE](#licence)
 
 <!-- /TOC -->
 
 ## Docker, Singularity and Dockstore
 
 There are pre-built images containing this codebase on quay.io.
 
+* [cgpPindel][cgpPindel-git]: Contained within this repository - contains the cgpPindel package
 * [dockstore-cgpwxs][ds-cgpwxs-git]: Contains tools specific to WXS analysis.
 * [dockstore-cgpwgs][ds-cgpwgs-git]: Contains additional tools for WGS analysis.
 
 These were primarily designed for use with dockstore.org but can be used as normal containers.
 
-The docker images are know to work correctly after import into a singularity image.
+The docker images know to work correctly after import into a singularity image.
```

Comment:
I think thats the wrong edit.  Pretty sure it should have read:

_The docker images **are known** to work correctly after import into a singularity image._

Reply:
Fixed.